src/Versioning/MinimalComponentHistoryModel.cs

```diff
@@ -64,13 +64,23 @@ public class MinimalComponentHistoryModel : MinimalComponentModel, IChangesetSna
         public bool ContentChanged { get; set; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether the component can be restored.
+        /// Gets a value indicating whether the component can be restored.
         /// </summary>
         public bool CanRestore => this.Action != ChangesetSnapshotActionType.Delete && !this.LatestSnapshot;
 
         /// <summary>
         /// Gets or sets a value indicating whether this component is the latest snapshot.
         /// </summary>
         public bool LatestSnapshot { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the user who esigned.
+        /// </summary>
+        public string EsignedBy { get; set; }
```

Comment:
I assume this is more like the created / update by field, don't we generally use user id mapping in that case? Please let me know if there are any specific reasons for capturing names directly.

Reply:
As discussed on Slack, closing this as CFR requirement.

Reply:
That's part of the CFR Part-11 requirement, we have to capture these info and store it forever.